behavenet/fitting/eval.py

```diff
@@ -461,6 +461,7 @@ def export_train_plots(hparams, dtype, loss_type='mse', save_file=None, format='
     import pandas as pd
     import seaborn as sns
     import matplotlib.pyplot as plt
+    mpl.use('Agg') #deal with display-less machines
```

Comment:
wait is `mpl` defined elsewhere?

Reply:
whoops sorry that's supposed to refer to matplotlib - just a sec